package.json

```diff
@@ -20,249 +19,228 @@
+          "Support for single quote escapes in Jinja"
         ],
-        "url": "https://raw.githubusercontent.com/ansible-community/schemas/main/f/molecule.json"
+        "path": "./syntaxes/ansible/generated/jinja-single-quote-escape.tmLanguage",
+        "scopeName": "injection.ansible.jinja-single-quote-escape",
+        "injectTo": [
+          "source.ansible"
+        ]
       },
       {
-        "fileMatch": [
-          "galaxy.yml"
+        "comments": [
+          "The quote escapes implemented in the conditionals injection do not work inside brackets etc.",
+          "Support for double quote escapes in Jinja"
         ],
-        "url": "https://raw.githubusercontent.com/ansible-community/schemas/main/f/ansible-galaxy.json"
+        "path": "./syntaxes/ansible/generated/jinja-double-quote-escape.tmLanguage",
+        "scopeName": "injection.ansible.jinja-double-quote-escape",
+        "injectTo": [
+          "source.ansible"
+        ]
       }
     ]
   },
-  "contributors": [
-    {
-      "email": "ssbarnea@redhat.com",
-      "name": "Sorin Sbarnea"
-    }
-  ],
   "dependencies": {
-    "@commitlint/cli": "latest",
-    "@commitlint/config-conventional": "latest",
-    "@types/ini": "^1.3.30",
-    "@types/tmp": "^0.2.0",
-    "ini": "^2.0.0",
-    "tmp": "^0.2.1",
-    "untildify": "^4.0.0",
-    "vscode-nls": "^5.0.0"
+    "ansible-language-server": "^0.1.1-0",
+    "vscode-languageclient": "^7.0.0"
   },
-  "description": "Ansible YAML schema verification, auto-complete, highlight problems reported by ansible-lint or yamllint and vault encryption and decryption.",
+  "description": "Ansible language support",
   "devDependencies": {
+    "@commitlint/cli": "^13.1.0",
+    "@commitlint/config-conventional": "^13.1.0",
+    "@types/chai": "^4.2.21",
     "@types/glob": "^7.1.3",
-    "@types/mocha": "^8.2.2",
-    "@types/node": "^15.0.2",
-    "@types/vscode": "^1.52.0",
-    "@typescript-eslint/eslint-plugin": "^4.23.0",
-    "@typescript-eslint/parser": "^4.23.0",
-    "eslint": "^7.26.0",
-    "glob": "^7.1.7",
+    "@types/mocha": "^8.2.3",
+    "@types/node": "^14.17.5",
+    "@types/vscode": "^1.56.0",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
+    "@typescript-eslint/parser": "^4.28.3",
+    "copyfiles": "^2.4.1",
+    "eslint": "^7.30.0",
+    "eslint-config-prettier": "^8.3.0",
     "husky": "^6.0.0",
     "mocha": "^8.4.0",
+    "rimraf": "^3.0.2",
     "standard-version": "^9.3.0",
-    "typescript": "^4.2.4",
-    "vsce": "^1.88.0",
-    "vscode-test": "^1.5.2"
+    "ts-loader": "^8.3.0",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.3.5",
+    "vsce": "^1.95.1",
+    "vscode-test": "^1.5.2",
+    "webpack": "^5.44.0",
+    "webpack-cli": "^4.7.2"
   },
-  "displayName": "Ansible Language",
+  "displayName": "Ansible",
   "engineStrict": true,
   "engines": {
     "node": ">=16.0",
     "npm": "^7.12.1",
-    "vscode": "^1.52.0",
+    "vscode": "^1.56.0",
```

Comment:
Was it really necessary to upgrade the minimum version of VS Code? Why?

Reply:
cc @ssbarnea do you know anything about this?

Reply:
That was needed for two reasons: some required by yaml language server (so no longer needed), the second was related to file extension patterns, which were fixed my microsoft only recently, older versions not being able to use the kind of patterns we needed for identifying playbooks or other ansible files. It may now be possible to lower that number.